### PR TITLE
GoogleDrive copy file command

### PR DESCRIPTION
**Pull request by jlevypaloalto**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-477)

## Description
Add the `google-drive-file-copy` command to the GoogleDrive integration to support copying files.

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/GoogleDrive/Integrations/GoogleDrive/GoogleDrive.yml [hunk 1]
```diff
@@ -135,17 +135,17 @@ script:
...
-        predefined:
-          - "True"
-          - "False"
+    - description: The user's primary email address.
+      name: user_id
+    - description: The name of this shared drive.
+      name: name
+      required: true
+    - auto: PREDEFINED
+      description: 'Whether the shared drive is hidden from the default view. Possible values: "True" and "False".'
```
Comment (ShirleyDenkberg) [inline Packs/GoogleDrive/Integrations/GoogleDrive/GoogleDrive.yml:144]:
```suggestion
      description: Whether the shared drive is hidden from the default view.
```

File: Packs/GoogleDrive/ReleaseNotes/1_2_44.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Google Drive
+
+- Added a new command: `google-drive-file-copy`.
```
Comment (ShirleyDenkberg) [inline Packs/GoogleDrive/ReleaseNotes/1_2_44.md:6]:
```suggestion
Added a new command: `google-drive-file-copy`.
```
```suggestion
- Added a new command: ***google-drive-file-copy*** that makes a copy of a Google Drive file.
```

**Comment by ShirleyDenkberg** [comment]
@JudahSchwartz Doc review completed.

File: Packs/GoogleDrive/ReleaseNotes/1_2_44.md (hunk 2 above)
Comment (jlevypaloalto) [reply Packs/GoogleDrive/ReleaseNotes/1_2_44.md:6]: Done

File: Packs/GoogleDrive/Integrations/GoogleDrive/GoogleDrive.yml (hunk 1 above)
Comment (jlevypaloalto) [reply Packs/GoogleDrive/Integrations/GoogleDrive/GoogleDrive.yml:144]: Done

File: Packs/GoogleDrive/Integrations/GoogleDrive/README.md [hunk 3]
```diff
@@ -1304,3 +1304,52 @@ Modify labels to file.
...
+        "mimeType": "application/vnd.google-apps.spreadsheet",
+        "name": "New Copy"
+    }
+}
+```
+
+#### Human Readable Output
+
+>### New file copied from *1O8Gx7DslVpbd-HN7lp4MIN1DDakpw-bHVHCwir2wUlo*
+>|Id|
```
Comment (JudahSchwartz) [inline Packs/GoogleDrive/Integrations/GoogleDrive/README.md:1352]: why only have id in readable output?

File: Packs/GoogleDrive/Integrations/GoogleDrive/GoogleDrive_test.py [hunk 4]
```diff
@@ -1003,3 +1003,46 @@ def test_upload_file_with_parent_command_success(self, mocker, gsuite_client):
...
+        - A request to copy a Drive file.
+
+        When:
+        - Calling google-drive-file-copy.
+
+        Then:
+        - Copy the Drive file.
+        """
+
+        from GoogleDrive import file_copy_command
```
Comment (JudahSchwartz) [inline Packs/GoogleDrive/Integrations/GoogleDrive/GoogleDrive_test.py:1019]: lets also mock a failure response, like when id not found

**Review by JudahSchwartz — COMMENTED** [review]
Awesome! lets do final review after the demo

File: Packs/GoogleDrive/Integrations/GoogleDrive/README.md (hunk 3 above)
Comment (jlevypaloalto) [reply Packs/GoogleDrive/Integrations/GoogleDrive/README.md:1352]: Keeping it consistent with the other googledrive functions.

Comment (jlevypaloalto) [reply Packs/GoogleDrive/Integrations/GoogleDrive/README.md:1352]: I added more info

**Review by JudahSchwartz — APPROVED** [review]
awesome!

**Pull request merged**